inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -185,7 +196,35 @@ def from_curves_file(cls,
 
         return cls(coils, num_field_periods)
 
-    def simulate(self):
+    def simulate(
+        self,
+        simulator: Simulator = Simsopt(),
+        resource_pool_id: Optional[UUID] = None,
+        run_async: bool = False,
+        magnetic_field_filename: str = 'magnetic_field.npz',
```

Comment:
What is this parameter supposed to control? If it's the name of the file created by the simulator on the backend, then this shouldn't be controllable from the client. That file has a default name defined by the executer (or the underlying "simulator").

Reply:
So I will remove it has a simulator argument and that means I will also need to remove self.args.magnetic_field_filenamen from the API, right?

Reply:
Yep. It can have a default name controlled by the API.